content/3.pecan/1.overview/2.methods-and-data.md

```diff
@@ -153,20 +153,20 @@ Example data from 790 tumor samples can be found in [Figure 5 of McLeod et al](h
 
 ### t-SNE Methods
 
-[RAPID RNA-Seq workflow](https://university.stjude.cloud/docs/genomics-platform/workflow-guides/rapid-rnaseq/) was used to generate RNA-seq expression count data and visualized via ProteinPaint's scatter plot.
+St. Jude Cloud's [RNA-Seq V2 WDL workflow](https://github.com/stjudecloud/workflows/blob/main/workflows/rnaseq/rnaseq-standard.wdl) was used to generate HTSeq counts data for each sample and visualized on PeCan's expression t-SNE plot.
```

Comment:
```suggestion
St. Jude Cloud's [RNA-Seq V2 WDL workflow](https://github.com/stjudecloud/workflows/blob/rnaseq-standard/v2.0.0/workflows/rnaseq/rnaseq-standard.wdl) was used to generate HTSeq counts data for each sample and visualized on PeCan's expression t-SNE plot.
```

As far as I'm aware Plaid is using the v2.0.0 release. So I think we should link to it directly. It adds some maintenance burden if we ever upgrade.